### PR TITLE
fix: Patch issue on Dockerfile missplaced

### PR DESCRIPTION
**Pull request by CodeWithEmad**
it's the fix for issue https://github.com/overhangio/tutor/issues/597

### CODE REVIEW

**Review by regisb — CHANGES_REQUESTED** [review]
Moving the patch is not acceptable, as it would break backward compatibility. See my comment for alternative solutions.

File: tutor/templates/build/openedx/Dockerfile [hunk 1]
```diff
@@ -210,6 +208,9 @@ FROM production as development
 
 # Install useful system requirements (as root)
 USER root
+
+{{ patch("openedx-dockerfile") }}
```
[the remark this answers is not in the document]
Comment (CodeWithEmad) [reply tutor/templates/build/openedx/Dockerfile:212]: Awesome solutions @regisb, like always. thanks for your time and effort put in the Tutor project

[the remark this answers is not in the document]
Comment (regisb) [reply tutor/templates/build/openedx/Dockerfile:212]: Will you open a PR to add a "openedx-dockerfile-minimal" patch?

[the remark this answers is not in the document]
Comment (CodeWithEmad) [reply tutor/templates/build/openedx/Dockerfile:212]: sure, on it :)

[the remark this answers is not in the document]
Comment (CodeWithEmad) [reply tutor/templates/build/openedx/Dockerfile:212]: please take a look at PR https://github.com/overhangio/tutor/pull/610

**Pull request closed without merging**